src/Core/Extensions/GraphQLBuilderCoreExtensions.cs

```diff
@@ -16,12 +17,45 @@ namespace GraphQL.Server
     /// </summary>
     public static class GraphQLBuilderCoreExtensions
     {
+        /// <summary>
+        /// Registers an instance of <see cref="BasicGraphQLExecuter{TSchema}"/> as <see cref="IGraphQLExecuter{TSchema}"/> for
+        /// use with <see cref="GraphQLOptions"/>. It is recommended to use <see cref="IDocumentExecuter"/> directly rather than
```

Comment:
> It is recommended to use

Why?

Reply:
If I remember right, the class really serves no purpose anymore, unless you want to add additional functionality in an abstract fashion by inheriting the class.  It just seemed pointless.  Originally this class pulled in middleware and document listeners via DI and applied them to the execution options.  But now all the configuration can be done with extension methods on `IGraphQLBuilder`.

I could be wrong; that's my memory from 6 months ago...

Reply:
https://github.com/graphql-dotnet/server/issues/679